sources.md

```diff
@@ -1,6 +1,6 @@
 # Sources
 
-Still incomplete. See https://github.com/vega/vega-datasets/issues/15
+Still incomplete. See <https://github.com/vega/vega-datasets/issues/15>
```

Comment:
Markdown on GitHub automatically detects links. I don't think this is necessary.